src/containers/FlightDirector/MacroButtons/macroConfig.js

```diff
@@ -104,6 +106,46 @@ const ActionList = ({
           )}
         </Mutation>
       </Label>
+      {/* SORTABLE ACTION LIST by Sam Hill */}
+      <Mutation
+        mutation={gql`
+          mutation ReorderActions(
+            $configId: ID!
+            $id: ID!
+            $oldIndex: Int!
+            $newIndex: Int!
+          ) {
+            reorderMacroAction(
+              configId: $configId
+              id: $id
+              oldIndex: $oldIndex
+              newIndex: $newIndex
+            )
+          }
+        `}
+      >
+        {action => (
+          <SortableActionList
+            css={css`
+              flex: 1;
+              overflow-y: auto;
+            `}
+            distance={20}
+            actions={actions}
+            id={id}
+            selectedAction={selectedAction}
+            setSelectedAction={setSelectedAction}
+            removeAction={removeAction}
+            onSortEnd={({oldIndex, newIndex}) =>
+              action({
+                variables: {configId, id, oldIndex, newIndex},
+              })
+            }
+          />
+        )}
+      </Mutation>
+
+      {/* Old macro action list
```

Comment:
If you've commented out all of the old code that you've replaced, you should probably just delete that code anyway.